openshift/topics/advanced_concepts.adoc

```diff
@@ -450,7 +516,7 @@ deploymentconfig "sso" scaled
 +
 [NOTE]
 ====
-In the preceding command, ``sso-mysql`` appears because a MySQL template was used to deploy the {project_openshift_product_name} image. However, if a PostgreSQL template was used, the name of the deployment config of the PostgreSQL pod would be ``sso-postgresql``.
+In the preceding command, ``sso-postgresql`` appears because a PostgreSQL template was used to deploy the {project_openshift_product_name} image. However, if a PostgreSQL template was used, the name of the deployment config of the PostgreSQL pod would be ``sso-postgresql``.
```

Comment:
Drop the second sentence starting from `However, ....` (current form uses postgresql example two times).